tests/test_registerutils.py

```diff
@@ -12,29 +12,180 @@
 # License along with this library.
 
 import inspect
+import glob
 import os
 import sys
 
 test_path = os.path.abspath(
     os.path.dirname(inspect.getfile(inspect.currentframe())))
 code_path = os.path.abspath('%s/../lib' % test_path)
 config_path = os.path.abspath('%s/../etc' % test_path)
+data_path = test_path + os.sep + 'data/'
```

Comment:
Could use os.path.join with a trailing empty string to get the correct sep based on os.